.gitignore

```diff
@@ -1,2 +1,3 @@
 bin
-kubebuilder
\ No newline at end of file
+ikubebuilder
```

Comment:
this must be `kubebuilder` not `ikubebuilder` here.

Reply:
corrected